### DISCUSSION TITLE
Decrease docs content font size from 18px to 16px

### DISCUSSION THREAD

**Discussion by visortelle** [apache/pulsar#22155] [Ideas]
It's a minor question, but probably resonates with someone who reads the docs often.

When I visit Pulsar docs, I often feel that something is wrong with the font size and want to press <kbd>CMD</kbd> + <kbd>-</kbd> to decrease it.

- GitHub uses 16px for markdown content
- Docusaurus uses 16px
- Reddit uses 14px
- Most sites on the Internet use 16px

I would also prefer to see more content on my laptop screen by default.

We discussed the font size almost a year ago with @emidio-cardeira and @asafm at the latest site redesign and decided to go with 18px for some reason.

I wonder, are there more people who also find 18px bigger than it should be?

The difference in content amount the user sees on the screen is quite noticeable in the tables:

**18px**

<img width="825" alt="Screenshot 2024-02-29 at 2 12 12 AM" src="https://github.com/apache/pulsar/assets/9302460/ff136217-e4f6-41b0-b32d-49f27375b05d">

https://pulsar.apache.org/docs/next/concepts-broker-load-balancing-concepts/

**16px**

<img width="826" alt="Screenshot 2024-02-29 at 2 13 03 AM" src="https://github.com/apache/pulsar/assets/9302460/87cdcfbf-3553-4b78-a8ba-1c973b60759d">

https://pulsar-site-2024-updates-git-docs-font-size-visortelle.vercel.app/docs/next/concepts-broker-load-balancing-concepts/

**Comment by asafm** [comment]
If most of the internet uses 16px, I'm ok with changing it back to 16px.
I'll personally hit CMD-+ :)

**Reply by visortelle** [reply]
@asafm, thank you :)
I'll apply the corresponding changes in a few days if there won't be objections.

**Comment by dave2wave** [comment]
16px is also a default for Apple Mail. +1.

**Comment by jak78** [comment]
+1Le 28 févr. 2024 à 23:27, Kiryl Valkovich ***@***.***> a écrit :﻿
It's a minor question, but probably resonates with someone.
When I visit Pulsar docs, I often feel that something is wrong with the font size and want to press CMD + - to decrease it.
GitHub uses 16px for markdown content. Docusaurus uses 16px. Most sites on the Internet use 16px.
I would also prefer to see more content on my laptop screen by default.
We discussed the font size almost a year ago with @emidio-cardeira and @asafm at the latest site redesign and decided to go with 18px for some reason.
I wonder, are there more people who also find 18px bigger than it should be?
The difference in content amount the user sees on the screen is quite noticeable in the tables:
18px
Screenshot.2024-02-29.at.2.12.12.AM.png (view on web)
https://pulsar.apache.org/docs/next/concepts-broker-load-balancing-concepts/
16px
Screenshot.2024-02-29.at.2.13.03.AM.png (view on web)
https://pulsar-site-2024-updates-git-docs-font-size-visortelle.vercel.app/docs/next/concepts-broker-load-balancing-concepts/

**Comment by visortelle** [comment]
I merged the PR: https://github.com/apache/pulsar-site/pull/814
